docs/data/material/guides/api/api.md

```diff
@@ -91,16 +91,23 @@ Nested components inside a component have:
 
 ### Prop naming
 
-The name of a boolean prop should be chosen based on the **default value**. This choice allows:
+- **Boolean**
 
-- the shorthand notation. For example, the `disabled` attribute on an input element, if supplied, defaults to `true`:
+  - The default value of a boolean prop should be `false`. This choice allows the shorthand notation For example an input is by default enabled: How to name the prop that controls this state? It should be called `disabled` because:
 
-  ```jsx
-  <Input enabled={false} /> ❌
-  <Input disabled /> ✅
-  ```
+    ```jsx
+    ❌ <Input enabled={false} />
+    ✅ <Input disabled />
+    ```
+
+  - The name of the boolean, if composed of a single word, should be an adjective or a noun and not a verb. This is because the prop describes a state. For example an input prop can be controlled by a state, which wouldn't be called with a verb:
```

Comment:
Grammar and a little more clarity.

```suggestion
  - If the name of the boolean is a single word, should be an adjective or a noun rather than a verb. This is because props describe a _state_ and not an _action_. For example an input prop can be controlled by a state, which wouldn't be described with a verb:
```